submissions/MutmainahOdunewu/html-forms/signup.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../css/signup.css">
+    <title>signup page</title>
+</head>
+<body>
+    <nav class="nav-bar">
+        <h2><i>sandalsbymo</i></h2>
+        <ul class="look">
+            <li><a href="../HTML/index.html">Product Page</a></li>
+            <li><a href="../html-forms/contact.html">Contact Page</a></li>
+            <li><a href="./signup.html"></a>Sign Up</li>
+        
+        </ul>
+    </nav>
+    <div class="container">
+        <div class="side" id ="side">
+            <h1>REGISTER YOUR ACCOUNT</h1>
+            <p>Registering an account gives you access to sweet promo and<br> events available to only registered customers</p>
+          <p>Already got an account?<span><a href="#">Sign in</a></span></p>
+        </div>
+        <div class="side">
+            <form>
+                <label for="firstname">First Name</label><br>
+          <input type="text" name="firstname" id="firstname"/><br><br>
+          <label for="lastname">Last Name</label><br>
+          <input type="text" name="lastname" id="lastname"/><br><br>
+          <label for="othername">Other Names</label><br>
+          <input type="text" name="othername" id="othername"/><br><br>
+          <label for="phonenumber">Phone Number</label><br>
+          <input type="text" name="phonenumber" id="phonenumber"/><br><br>
+          <label for="emailaddress">Email Address</label><br>
+          <input type="email" name="emailaddress" id="emailaddress"/><br><br>
+          <label for="password">Password</label><br>
+          <input type="password" name="password" id="password"/><br><br>
+          <label for="confirm">Confirm Password</label><br>
+          <input type="password" name="confirm" id="confirm"/><br><br>
+          <input type="checkbox" name="agree" id="agree" value="agree"/>
+          <label for="agree" id="lab">I agree to the Terms and Conditions</label><br><br>
+          <button type="submit" id="bttn">SIGN UP</button>
```

Comment:
This should be under the form tag, format properly. Also, remove the breaks and style with css instead.